pkg/misc/time_test.go

```diff
@@ -0,0 +1,26 @@
+package misc
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
```

Comment:
pls format this import block, group stdlib imports, group the third party imports.

Reply:
man, now, i can not configure my `.gitconfig` file for multilpe account for gitlab and github

Reply:
you can config `~/.ssh/config`:
```
Host github.com
    HostName github.com
    IdentityFile ~/.ssh/id_rsa_github
```

Reply:
i have those on my `~/.ssh/config`
```
# GitLab.com
Host gitlab.com
  PreferredAuthentications publickey
  IdentityFile ~/.ssh/id_ed25519

# Personal github account
Host github.com
   HostName github.com
   IdentityFile ~/.ssh/id_rsa_github
```

Reply:
> you can config `~/.ssh/config`:
> 
> ```
> Host github.com
>     HostName github.com
>     IdentityFile ~/.ssh/id_rsa_github
> ```

for mine like this, but i got error from github actions, sorry for my bad, this one is first time for me